### PR TITLE
Issue#849

### PR DESCRIPTION
**Pull request by reactexcel**

### CODE REVIEW

File: frontend/src/components/Task/Task.js [hunk 1]
```diff
@@ -15,22 +13,28 @@ class Task extends PureComponent {
...
+        <div className="form-check abc-checkbox abc-checkbox-success mb-sm" id ='top'>
           <input
-            className="form-check-input"
+            className="custom-checkbox form-check-input"
             id={`checkbox${id}`}
             checked={status}
             type="checkbox"
-            onChange={() => {
+            // onChange={() => {
+            //   handleChangeSteps(id);
```
Comment (xmagestore) [inline frontend/src/components/Task/Task.js]: remove commented code

File: frontend/src/modules/manageUsers/components/BasicDetail.js [hunk 2]
```diff
@@ -151,14 +151,14 @@ export default class BasicDetail extends Component {
...
         return (
             <div>
-            <h4>Personal Details</h4>
+            {/* <h4>Personal Details</h4>
             <br />
             <div className="row no-gutter">
               <div className="col-md-6">
                 Employee Id :<b> {this.state.user_id}</b>
                 <br />
                 Username :<b> {this.state.username}</b>
```
Comment (xmagestore) [inline frontend/src/modules/manageUsers/components/BasicDetail.js]: remove commented code

File: frontend/src/modules/manageUsers/containers/manageUsers.jsx [hunk 3]
```diff
@@ -455,185 +432,278 @@ class ManageUsers extends React.Component {
...
+                      this.toggleSecondTabs("tab21");
+                    }}
+                  >
+                    <span>Basic Detail</span>
+                  </NavLink>
+                </NavItem>
+                <NavItem>
+                  <NavLink
+                    className={classnames({
+                      active: this.state.activeSecondTab === "tab22"
```
Comment (xmagestore) [inline frontend/src/modules/manageUsers/containers/manageUsers.jsx]: what is all this tab22 21 etcc etc

Comment (reactexcel) [reply frontend/src/modules/manageUsers/containers/manageUsers.jsx]: sir there are multiple tabs so we are using tab id to switch between tabs.

**Pull request merged**